security/pkg/server/ca/authenticate/authenticator.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 const (
-	bearerTokenPrefix = "Bearer "
-	httpAuthHeader    = "authorization"
-	idTokenIssuer     = "https://accounts.google.com"
+	bearerTokenPrefix   = "Bearer "
+	httpAuthHeader      = "authorization"
+	httpClusterIDHeader = "clusterid"
```

Comment:
Do we want to pass it in a HTTP header ? I guess it's a quick hack, but most APIs put this in request.

Reply:
no，fixed